lib/cocoapods/target/build_settings.rb

```diff
@@ -592,6 +593,9 @@ def initialize(target, non_library_spec = nil, configuration: nil)
         # @return [Array<String>]
         define_build_settings_method :dynamic_frameworks_to_import, :memoized => true do
           dynamic_frameworks_to_import = vendored_dynamic_frameworks.map { |f| File.basename(f, '.framework') }
+          dynamic_frameworks_to_import.concat vendored_xcframeworks
```

Comment:
Shouldn't there be similar code for `:static_frameworks_to_import`?

Reply:
Yes, for this I'm starting with dynamically linked frameworks and will do a follow-up for static frameworks

Reply:
Went ahead and added support for this in this PR